### PR TITLE
London 9 - Mo Mahboobian - JS-Core-1 - Week 2

### PR DESCRIPTION
**Pull request by momahboobian**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Mo Mahboobian
- Your City: London
- Your Slack Name: Mo Mahboobian

# Homework Details

- Module: JS-Core-1
- Week: Week 2

# Notes

- What did you find easy?
Excersises
- What did you find hard?
No 3 of mandatory 
- What do you still not understand?
How to not execute a program and guessing the output
- Any other notes?
I couldn't run any testes 1 by 1. The package.js was testing all mandatory exercises together!

### CODE REVIEW

**Comment by momahboobian** [comment]
@qingwei91 
Hellow Qing, Really appreciated if you can code review my coursework. Thank you

File: extra/1-radio-stations.js [hunk 1]
```diff
@@ -14,7 +14,13 @@
  */
 
 // `getAllFrequencies` goes here
-
+function getAllFrequencies() {
+  let frequencies = [];
```
Comment (qingwei91) [inline extra/1-radio-stations.js:18]: Well done 👍 

If you're free, you can try to find a different way to do this, without using loop

File: extra/1-radio-stations.js [hunk 2]
```diff
@@ -25,7 +31,16 @@
  * - Return only the frequencies that are radio stations.
  */
 // `getStations` goes here
-
+function getStations() {
```
Comment (qingwei91) [inline extra/1-radio-stations.js:34]: This looks good too!

Another tip, you can try to use `filter` to implement this

File: mandatory/3-playing-computer.js [hunk 3]
```diff
@@ -28,6 +28,10 @@ const f2 = function (a, b) {
 
 console.log(x);
 console.log(a);
+<<<<<<< HEAD
+//console.log(b);
+=======
+>>>>>>> 8a8564fefa1deece60b24d5b10fa131be32d8a80
```
Comment (qingwei91) [inline mandatory/3-playing-computer.js:34]: Looks like you have a merge conflict you forgot to remove?

This is what you get when there's a merge conflict, and you need to manually resolve it by choose the code you want to keep. You can also remove all of them

**Review by qingwei91 — APPROVED** [review]

File: extra/1-radio-stations.js (hunk 2 above)
Comment (momahboobian) [reply extra/1-radio-stations.js:39]: @qingwei91 Still cant figure it out how to not lopp.
Maybe I didn't understand your meaning.
Can you help me with that please? 😊
Thanks

**Review by momahboobian — COMMENTED** [review]
@qingwei91  Can you Please check my extra exercise
Thanks

File: extra/1-radio-stations.js (hunk 2 above)
Comment (qingwei91) [reply extra/1-radio-stations.js:39]: Hi @momahboobian , you can read about this method, then it should be clearer: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter

**Review by qingwei91 — APPROVED** [review]

**Pull request closed without merging**